assets/thanos-ruler/thanos-ruler.yaml

```diff
@@ -76,8 +60,10 @@ spec:
     volumeMounts:
     - mountPath: /etc/tls/private
       name: secret-thanos-ruler-tls
-    - mountPath: /etc/proxy/secrets
-      name: secret-thanos-ruler-oauth-cookie
+      readOnly: true
+    - mountPath: /etc/kube-rbac-proxy
```

Comment:
I don't know if there is a mechanism to "reload" the proxies if such secrets change without the deployments being changed. (or if we need to pair them, it's not as critical as the certs as we'll be making the changes thus we'll test them, but we may overlook things)

Reply:
(not related to this PR, we can discuss/address this later)

Reply:
kube-rbac-proxy should live-reload certificates

Reply:
yep, I was talking about the proxy config itself.
Whether changing `thanos-user-workload-kube-rbac-proxy-web` in the future requires a manual reloading of the proxy or if we should add a hash or sth as we do for some certs for others components.

Reply:
NP, I'll create a ticket for that.